angular-frontend/AUTHORIZATION.md

```diff
@@ -0,0 +1,81 @@
+# Authorization
+To leverage the benefits of Tanzu AppSSO Authserver, there are a few things you should take into consideration.
+
+## Inner Loop (Development Mode)
+### Configuration of auth.conf.json
+Your Angular app should have an `src/assets/auth.conf.json` file that needs configuration. The items in the file correspond to the following:
+
+```json
+{
+  "clientId": "your-client-id",
+  "authority": "https://your-authorization-server.com/",
+  "scope": [
+    "openid" // This list MUST contain openid
+  ]
+}
+```
+
+__Note:__ The scpoes listed here are default scopes. Replace with the scopes appropriate for your application. Any scopes you 
+want to have available here also need to be listed in the `clientregistration.yaml`
+
+## Authserver Configuration
+To enable a single page app with an AppSSO Authserver, you will need to implement CORS. You can specify CORS settings in the AuthServer CR spec.cors section. Here are some examples:
```

Comment:
**[suggestion]**: I wouldn't use the wording "implement CORS", but maybe "you will need to define allowed HTTP origins, as part of CORS"

**[suggestion]**: AuthServer should be in back ticks (`AuthServer`), and `spec.cors` as well.